code/game/gamemodes/cult/talisman.dm

```diff
@@ -147,6 +147,11 @@
 		return ..(user, 0)
 
 	var/input_rune_key = input(user, "Choose a rune to teleport to.", "Rune to Teleport to") as null|anything in potential_runes //we know what key they picked

```

Comment:
```
	var/input_rune_key = input(user, "Choose a rune to teleport to.", "Rune to Teleport to") as null|anything in potential_runes //we know what key they picked
	var/obj/effect/rune/teleport/actual_selected_rune = potential_runes[input_rune_key] //what rune does that key correspond to?
	if(!src || qdeleted(src) || !user || user.l_hand != src && user.r_hand != src || user.incapacitated() || !actual_selected_rune)
		return ..(user, 0)
```

99% of the time you don't want to check for `stat || restrained()`

Reply:
True but one of the issues here is that if you open the input box and then die you can still teleport. If a cultist was using an anti drop implant they could still do so while dead.

Reply:
`incapacitated` checks for that.

Reply:
👍